chapters/1.md

```diff
@@ -11,33 +11,39 @@ Maria is purchasing a car from George, and in this chapter we will consider thre
+We are in an interconnected world. Natural disasters, epidemics, decisions made by one government, a company or even a single person can have cascading effects that lead to significant economic disruption worldwide. The 2008 global financial crisis played an important role in demonstrating the need for a more robust financial system.  People began experimenting and thinking about alternatives to the current banking system. Using cryptography to build a better money system was not a new idea, but it had hit a dead end and most researchers had given up hope it could be made trustless.
 
-Thankfully, an emerging new blockchain technology is capable of mitigating all of the above risks by creating a distributed ledger that all parties can equally use, view, and verify. This remarkable capability for strangers to agree on a shared document, which is called decentralized consensus, has been revolutionized in the last decade.
+The overarching challenge was to create a completely verifiable monetary system without needing to trust a central third party. By decentralizing power over issuance and transaction processing, it was theorized that the economic participants could collectively keep each other honest and resist attempts at manipulation.
 
-It is easy to be confused about the terminology at first, especially since most people are simultaneously introduced to several jargony concepts. You can think about "blockchains" as a technology that allows networks to establish "decentralized consensus" agreements. By enabling strangers to safely share a ledger, it becomes possible to build "cryptocurrencies" that function as digital cash. There are a multitude of regular currencies (euros, dollars, yen, etc); analogously, various teams have built many different cryptocurrencies (Monero, Ethereum, Bitcoin, etc).
+One significant obstacle in creating such a system was that there was no known way to prevent people from spending the same money twice without having a central party to validate each transaction.  Given that some network participants may try to lie, cheat, pretend to be multiple people, etc, how do you maintain a single version of the ledger to prevent double spending?  In computing, this is illustrated in a thought experiment referred to as the “Byzantine Generals problem” meant to demonstrate the pitfalls and design challenges of attempting to coordinate an action by communicating over an unreliable network. In the experiment, two generals are only able to communicate with one another by sending a messenger through enemy territory. The experiment asks how they might reach an agreement on the time to launch an attack, while knowing that any messenger they send could be captured.
```

Comment:
> while knowing that any messenger they send could be captured.

Byzantine General problem does not face the Man-in-the-middle problem, but instead the "deadlock problem". The main idea of that paragraph is to convey the message that any general should base own decision on the messages they receive.

If you have two generals A,B that send two different messages (e.g. "defend" and "attack"), the general C will have a lot of difficulties on choosing, since 1/2 is "defend" and 1/2 is "attack" (so 50% 50%, general C can not decide). The stall (a sort of "deadlock") described by this paragraph is the **exceptional case** that any parallel decentralized system should avoid. 

I have mixed feelings about mentioning this problem on first chapter: it's a part of theory of distributed systems and I do not want to create any headache to reader. However, it's still a central point of cryptocurrencies.

Reply:
> I have mixed feelings about mentioning this problem on first chapter

I do too... I was just rewording it because it was already there.  Much of it came from Wikipedia [Two_Generals' Problem](https://en.wikipedia.org/wiki/Two_Generals%27_Problem).  Shall I remove it?

Reply:
I think it might be better to dig into this problem in the new technical chapter about blockchain.

A suggestion: do not take a look for any website while writing. Our mind can remember what we read and I do not want the content might sound too much similar to an article, or a wiki.

Reply:
Okay, removed.